compiler-plugin/src/main/kotlin/com/google/devtools/ksp/processing/impl/ResolverImpl.kt

```diff
@@ -880,22 +888,13 @@ class ResolverImpl(
 
     @KspExperimental
     override fun getJvmName(accessor: KSPropertyAccessor): String? {
-        val descriptor = resolvePropertyAccessorDeclaration(accessor)
-
-        return descriptor?.let {
-            // KotlinTypeMapper.mapSignature always uses OwnerKind.IMPLEMENTATION
-            typeMapper.mapFunctionName(descriptor, OwnerKind.IMPLEMENTATION)
-        }
+        return resolvePropertyAccessorDeclaration(accessor)?.let(typeMapper::mapFunctionName)
```

Comment:
This (and below) currently failed to build:
```
> Task :compiler-plugin:compileKotlin FAILED
e: file://.../ksp/compiler-plugin/src/main/kotlin/com/google/devtools/ksp/processing/impl/ResolverImpl.kt:891:66 Type mismatch: inferred type is KFunction2<FunctionDescriptor, OwnerKind?, String> but (TypeVariable(T)) -> TypeVariable(R) was expected
e: file://.../ksp/compiler-plugin/src/main/kotlin/com/google/devtools/ksp/processing/impl/ResolverImpl.kt:897:86 Type mismatch: inferred type is KFunction2<FunctionDescriptor, OwnerKind?, String> but (TypeVariable(T)) -> TypeVariable(R) was expected
```
but that's because you didn't upgrade kotlin version: https://github.com/google/ksp/blob/main/gradle.properties#L4

Why don't you change that line to, e.g., `2.1.20-dev-5634`?

I found that upgrading AA (`aaKotlinBaseVersion`) causes other problems, but those should be addressed separately.